esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -3833,16 +3834,49 @@ void CWsWuFileHelper::createWUZAPFile(IEspContext& context, IConstWorkUnit* cwu,
     createProcessLogfile(cwu, winfo, "Thor", folderToZIP.str());
     if (request.includeThorSlaveLog.isEmpty() || strieq(request.includeThorSlaveLog.str(), "on"))
         createThorSlaveLogfile(cwu, winfo, folderToZIP.str());
+#else
+    findPostMortemFolder(request.wuid, postMortemFolder);
```

Comment:
hm, this is not the way I expected it to work..
The folder should already be named in the workunit by the postmortem process..
See the original PR that added the postmortem mechanism: https://github.com/hpcc-systems/HPCC-Platform/pull/15732  (wutool postmortem ..)

Reply:
Will do some research to see why the folder (name and path) is not in WU.